app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,68 @@
 
     {%- endif %}
 
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "ScholarlyArticle",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
+            },
+            "headline": "{{ item.fullTitle|striptags }}",
+            "datePublished": "{{ item.publishedDate|date('Y-m-d', 'UTC') }}",
+            "author": [
+                {% for author in item.authors %}
+                    {% if author.orcid is defined %}
+                        {
+                            "@type": "Person",
+                            "name": "{{ author.preferredName }}"
+                        }
+                    {% elseif author.people is defined %}
+                        {
+                            "@type": "Organization",
+                            "name": "{{ author.toString() }}"
+                        }
+                    {% else %}
+                        "{{ author.toString() }}"
+                    {% endif %}
+                    {%- if not loop.last %}, {% endif -%}
+                {% endfor %}
+            ],
+            "publisher": {
+                "@type": "Organization",
+                "name": "eLife Sciences Publications, Ltd",
+                "logo": {
+                "@type": "ImageObject",
+                "logo": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
```

Comment:
`logo` isn't a property of `ImageObject`.

Prefer

```json
"publisher": {
    "@type": "Organization",
    "name": "eLife Sciences Publications, Ltd",
    "logo": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
}
```

which is a shortened version of:

```json
"publisher": {
    "@type": "Organization",
    "name": "eLife Sciences Publications, Ltd",
    "logo": {
        "@type": "ImageObject",
        "url": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
    }
}
```